uber/models.py

```diff
@@ -412,6 +430,8 @@ def apply(self, params, *, bools=(), checkgroups=(), restricted=True, ignore_csr
             if not ignore_csrf:
                 check_csrf(params.get('csrf_token'))
 
+        return self
```

Comment:
This is a super convenient change. `apply()` now returns `self`, so it the result can be used without needing a temporary variable.

So you can do this:
```
attendee.promo_code = PromoCode().apply(dict(code='code'))
```

instead of:
```
attendee.promo_code = PromoCode()
attendee.promo_code.apply(dict(code='code'))
```